test/stability-tests/src/it/java/software/amazon/awssdk/stability/tests/s3/S3BaseStabilityTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.stability.tests.s3;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import software.amazon.awssdk.http.apache.ApacheHttpClient;
+import software.amazon.awssdk.http.nio.netty.NettyNioAsyncHttpClient;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.s3.S3AsyncClient;
+import software.amazon.awssdk.services.s3.S3Client;
+import software.amazon.awssdk.services.s3.model.DeleteBucketRequest;
+import software.amazon.awssdk.testutils.service.AwsTestBase;
+import software.amazon.awssdk.utils.Logger;
+
+public abstract class S3BaseStabilityTest extends AwsTestBase {
+    private static final Logger log = Logger.loggerFor(S3BaseStabilityTest.class);
+    public static final int CONCURRENCY = 100;
+    public static final int TOTAL_REQUEST_NUMBER = 5_000;
+
+    protected static S3AsyncClient s3NettyClient;
+    protected static S3Client s3ApacheClient;
+
+    static {
+        s3NettyClient = S3AsyncClient.builder()
+                                     .httpClientBuilder(NettyNioAsyncHttpClient.builder()
+                                                                               .maxConcurrency(CONCURRENCY))
+                                     .credentialsProvider(CREDENTIALS_PROVIDER_CHAIN)
+                                     .region(Region.US_WEST_2)
```

Comment:
Any reason not to just use the let the default chain use the `AWS_REGION` value?

Reply:
I think there are some issues with default chain when running on Travis, but since this test is not running on Travis, I guess it's fine.